### ISSUE TITLE
Lisää koko maa avaimeen

### ISSUE THREAD

**Issue by jhuovari** [pttry/statficlassifications#3]
Yleensä kai SSS

Voisi ehkä olla optiona lisäys?

**Comment by jalasalmi** [comment]
Ai että avaimen kaikille riveille tulisi koko maa? Olisiko sen avulla aggregointi koko maan tasolle kätevämpää?

**Comment by jhuovari** [comment]
Ei, kun koko maalle oma rivi. Niin, että jos haluaa pitää mukana  koko maan summaa, niin se ei häviä aggrgoinnissa.
